### PR TITLE
The default service account will always be present when the namespace is created, so it will always return true

### PR DESCRIPTION
**Pull request by kahirokunn**
I created this to improve this issue.
https://github.com/weaveworks/eksctl/issues/4522

### CODE REVIEW

**Comment by kahirokunn** [comment]
Okay!
Just moment please!

**Comment by Skarlso** [comment]
@kahirokunn Hello. :)

Just asking if you are planning on adding the suggested tests? :)

**Comment by kahirokunn** [comment]
I've been so busy lately that I haven't had much time 😭.

I'll try to write about it when I have time, but others are welcome to write about it.

**Comment by Skarlso** [comment]
There is no rush! I'm just checking in. :) Thanks for getting back!

**Comment by Himangini** [comment]
@kahirokunn shinnen omedetô 🥳  Just checking on how's it going for this PR. Ping us if you need help. I am removing the review requests on the PR, please tag us again when the PR is ready for review 👍🏻

File: pkg/ctl/create/iamserviceaccount_test.go [hunk 1]
```diff
@@ -10,62 +12,65 @@ import (
...
-			count := 0
-			cmdutils.AddResourceCmd(cmdutils.NewGrouping(), cmd.parentCmd, func(cmd *cmdutils.Cmd) {
-				createIAMServiceAccountCmdWithRunFunc(cmd, func(cmd *cmdutils.Cmd, overrideExistingServiceAccounts bool) error {
-					Expect(cmd.ClusterConfig.Metadata.Name).To(Equal("clusterName"))
-					Expect(cmd.ClusterConfig.IAM.ServiceAccounts[0].Name).To(Equal("serviceAccountName"))
-					Expect(cmd.ClusterConfig.IAM.ServiceAccounts[0].AttachPolicyARNs).To(ContainElement("dummyPolicyArn"))
-					count++
-					return nil
+	saNames := [2]string{"sa-1", "default"}
+	for i := range saNames {
```
Comment (Skarlso) [inline pkg/ctl/create/iamserviceaccount_test.go:16]: Why the refactor of this DescribeTable?

I think you meant to add an `Entry` to this?

**Comment by kahirokunn** [comment]
I'm sorry, but due to my lack of time, I'm going to close this PR for now.
I hope that someone else will be able to accomplish this issue again.

**Pull request closed without merging**